migrations/sql/V2022.02.01.12.46__update_project_summary_status_codes.sql

```diff
@@ -0,0 +1,31 @@
+ALTER TABLE project_summary_status_code add column alias_description character varying(100);
+
+UPDATE project_summary_status_code SET active_ind = FALSE WHERE project_summary_status_code = 'OPN';
+UPDATE project_summary_status_code SET alias_description = 'Open' WHERE project_summary_status_code = 'OPN';
+
+UPDATE project_summary_status_code SET active_ind = FALSE WHERE project_summary_status_code = 'CLD';
+UPDATE project_summary_status_code SET alias_description = 'Closed' WHERE project_summary_status_code = 'CLD';
+
+UPDATE project_summary_status_code SET alias_description = 'Draft' WHERE project_summary_status_code = 'DFT';
+UPDATE project_summary_status_code SET alias_description = 'Withdrawn' WHERE project_summary_status_code = 'WDN';
+
+INSERT INTO project_summary_status_code (
+    project_summary_status_code, 
+    description, 
+    alias_description,
+    display_order, 
+    active_ind, 
+    create_user, 
+    update_user
+    )
+VALUES 
+    ('ASG', 'Assigned', 'Submitted', 50, true, 'system-mds', 'system-mds'),
+    ('UNP', 'Under review - with proponent', 'Under review', 60, true, 'system-mds', 'system-mds'),
+    ('UNR', 'Under review - with reviewers', 'Under review', 70, true, 'system-mds', 'system-mds'),
```

Comment:
Are these ever shown in the same dropdown using the alias vs the description? If so how do you choose the correct status since they have the same alias?

Reply:
There are two different dropDownSelectors - One for Core with the description as the label, and one in MineSpace as the alias_description as the label.